.circleci/continue_config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
                 name: Bundle Install if cache isn't present.
                 command: |
                   cd services/QuillLMS
-                  gem install bundler:2.2.33
+                  gem install bundler:2.3.17
```

Comment:
Do we need this line given the `Gemfile.lock` ends in the below? Maybe we can just remove this line rather than editing it so that the config is set in one place and can't drift?
```
BUNDLED WITH
   2.3.17
```